client/application/caproomster.constant.js

```diff
@@ -0,0 +1,11 @@
+(function(angular) {
+
+  'use strict';
+
+  angular.module('caproomster').constant('caproomster.AppConstant', {
+
+    server_base_url: 'http://127.0.0.1:5000' // change to to PROD server when finish
```

Comment:
I doubt we'll be actually hosting this on a VPS. I think for now, let's assume it will always be local :).